Tiltfile

```diff
@@ -36,6 +29,13 @@ os.putenv("CAPI_KIND_CLUSTER_NAME", settings.get("kind_cluster_name"))
 
 allow_k8s_contexts(settings.get("allowed_contexts"))
 
+if str(local("command -v " + kubectl_cmd + " || true", quiet = True)) == "":
```

Comment:
Apparently `allow_k8s_contexts` has to be run above "local" otherwise the local calls fail if the context is not a kind cluster or something.

(ran into this issue with a cluster created via CAPV)